ui/articles-aem-renderer/src/main/java/org/godtools/articles/aem/model/Article.java

```diff
@@ -0,0 +1,50 @@
+package org.godtools.articles.aem.model;
+
+import android.arch.persistence.room.ColumnInfo;
+import android.arch.persistence.room.Entity;
+import android.arch.persistence.room.PrimaryKey;
+import android.support.annotation.NonNull;
+
+/**
+ * This class represents the article table
+ *
+ * @author Gyasi Story
+ */
+@Entity(tableName = "article_table")
+public class Article {
+    /**
+     * Unique Identifier for the article table
+     */
+    @PrimaryKey(autoGenerate = true)
+    @NonNull
+    @ColumnInfo(name = "_id")
+    public int mId;
+
+    /**
+     * The title of the Article
+     */
+    @NonNull
+    @ColumnInfo(name = "title")
+    public String mTitle;
+
+    /**
+     * The main Content of the Article
+     */
+    @NonNull
+    @ColumnInfo(name = "content")
+    public String mContent;
+
+    /**
+     * The date the article was created
+     */
+    @NonNull
+    @ColumnInfo(name = "date_created")
+    public String mDateCreated;
```

Comment:
I know you had mentioned data transformers for dates. It would be great if on the POJO this could be a `Date` field, and it be stored as a `long` within the database.